extensions/wikia/WikiaMaps/WikiaMaps.i18n.php

```diff
@@ -1335,8 +1301,6 @@
 	'wikia-interactive-maps-create-map-choose-tile-set-tip' => '選擇現有的地圖模板或上傳新的地圖。',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "選擇現有的地圖模板或上傳新的地圖。" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)